xbmc/cores/VideoPlayer/DVDCodecs/Video/DXVA.h

```diff
@@ -73,7 +72,8 @@ class CRenderPicture
 public:
   CRenderPicture(CSurfaceContext *context);
   ~CRenderPicture();
-  ID3D11View*      view;
+  ID3D11View* view;
+  unsigned int extFormat;
```

Comment:
not sure if I get this correctly. but are are invited to do the change yourself. the source of this pr is a branch in xbmc repo. push the change to this branch. we can do cleanup/squashing later